src/imitation/scripts/train_adversarial.py

```diff
@@ -167,6 +167,7 @@ def callback(round_num: int, /) -> None:
     return {
         "imit_stats": imit_stats,
         "expert_stats": rollout.rollout_stats(expert_trajs),
+        "mean_return": imit_stats["monitor_return_mean"],
```

Comment:
Unclear why you set `mean_return` to `monitor_return_mean` here. Can we at least document this quirk of the `eval_policy` output? We also seem to do it in `train_rl.py` and `train_preference_comparisons.py`. So maybe add a function that just does this weird copying and document the reason in it's docstring to make this explicit.

Reply:
Removed the need to do all of this. Earlier, I used this hack to have a single location in the returned dict for the metric to be specified in `ray.tune.run`. Simplified this by using the original location of the metric for the different algorithms.